cypress/support/commands.js

```diff
@@ -24,18 +24,30 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
-
-addMatchImageSnapshotCommand();
-
 Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
```

Comment:
The `getByDataCy` command should return the result of `cy.get` to allow chaining of Cypress commands. Consider adding `return` before `cy.get` to enable this functionality.